NeuralAmpModeler/NeuralAmpModeler.cpp

```diff
@@ -438,6 +440,16 @@ NeuralAmpModeler::NeuralAmpModeler(const InstanceInfo& info)
     };
     toneStackSlider->SetActionFunction(toneStackAction);
 
+    // toggle IR on / off
+    IBSwitchControl* toggleIRactive = new IBSwitchControl(IRECT(46.f, 343.f, irSwitch), irSwitch, kIRToggle);
```

Comment:
Nit: Not a fan of the magic numbers. I'd prefer to see them defined as variables so it's clearer what they're doing.